### PR TITLE
Fix how dates are displayed in the events section in regard to local timezones

### PR DESCRIPTION
**Pull request by Jonas00000**
Fix of issue #86

### CODE REVIEW

**Comment by TimothyGillespie** [comment]
Looks good to me. I checked it out locally and verified it. I did notice, when a endTime is given, but not endDate the endtime gets ignored. That seems to make sense to me though and has been the previous behavior as well.

I don't see an error as of now here.

@Pixelguymm you may want to check it out, since it's also a partial self-review being that I gave some advised/opinion on this during the procses.

**Review by TimothyGillespie — APPROVED** [review]

File: src/lib/EventCards.svelte [hunk 1]
```diff
@@ -20,32 +20,32 @@
...
-        finalDateText += ` | ${startTimeText} UTC`
-      }
-      if (endDateText) {
-        finalDateText += ` - ${endDateText}`
-        if (endTimeText) {
-          finalDateText += ` | ${endTimeText} UTC`
-        }
+    let finalDateText = startDateText
+    if (startTimeText) {
+      const d = new Date()
```
Comment (Pixelguymm) [inline src/lib/EventCards.svelte:42]: This goes unused.

File: src/lib/EventCards.svelte [hunk 2]
```diff
@@ -20,32 +20,32 @@
...
+    const endDateText = endTimeUTC
+      ? endDate.toLocaleDateString('en-US', options)
+      : new Intl.DateTimeFormat('en-US', options).format(endDate)
+    const startTimeText = startTimeUTC
+      ? startDate.toLocaleTimeString('en-US', { hour: 'numeric', minute: 'numeric' })
       : null
-    const endTimeText = endTime
-      ? endTime.toLocaleTimeString(undefined, { hour: 'numeric', minute: 'numeric' })
+    const endTimeText = endTimeUTC
+      ? endDate.toLocaleTimeString('en-US', { hour: 'numeric', minute: 'numeric' })
```
Comment (Pixelguymm) [inline src/lib/EventCards.svelte:37]: I'm not sure if `en-US` is really the best locale to use for these. AM/PM can be confusing to international users (guilty as charged), so I think the 24-hour clock is the most straight-forward option.

**Review by Pixelguymm — CHANGES_REQUESTED** [review]

File: src/lib/EventCards.svelte (hunk 2 above)
Comment (Jonas00000) [reply src/lib/EventCards.svelte]: I used undefined now, this will result in the local time format being used which should be the best for everyone

**Review by Pixelguymm — APPROVED** [review]

**Pull request merged**